lib/virtfs/xfs/file_class.rb

```diff
@@ -0,0 +1,224 @@
+module VirtFS::XFS
+  class FS
```

Comment:
@jerryk55 As per my email. The FS class is composed of a number of files, containing the implementations of the class and instance methods of the File and Dir classes - all of which are instance methods of the FS. Instead of lumping them in one big file, I placed them in files based on those categories. To be consistent with my original prototype however, this file should probably be named `fs_file_class.rb`.